docs/reference/release-notes/highlights-6.4.0.asciidoc

```diff
@@ -7,3 +7,27 @@
 coming[6.4..0]
 
 See also <<release-notes-6.4.0,{es} 6.4.0 release notes>>. 
+
+=== Aggregations
+
+* Auto-interval Date Histogram - A new `auto_date_histogram` aggregaiton has been added which instead of taking an `interval` takes a `buckets` option which defines the maximum number of buckets it should return. The aggregation internally determines the best interval to use to get as close to the `bucket` option as possible without exceeding it. (https://github.com/elastic/elasticsearch/pull/28993)
```

Comment:
It would be nice to add a link to the documentation (e.g. https://www.elastic.co/guide/en/elasticsearch/reference/master/search-aggregations-bucket-autodatehistogram-aggregation.html), though I don't see that page in the 6.x version yet.

Reply:
The auto-interval date histogram has not quite been backported to 6.x yet. @pcsanwald is working on back porting it and I'll add this link when that is done